pkg/vm/engine/tae/containers/utils.go

```diff
@@ -122,3 +126,201 @@ func CopyToMoVectors(vecs []Vector) []*movec.Vector {
 	}
 	return movecs
 }
+
+func ProtoVectorToTAE(v *api.Vector, nullable bool) Vector {
+	moV, err := movec.ProtoVectorToVector(v)
+	if err != nil {
+		panic(err)
+	}
+	taeV := MOToTAEVector(moV, nullable)
+	return taeV
+}
+
+func MOToTAEVector(v *movec.Vector, nullable bool) Vector {
+	vec := MakeVector(v.Typ, nullable)
+	bs := NewBytes()
+	switch v.Typ.Oid {
+	case types.T_bool:
+		if v.Col == nil || len(v.Col.([]bool)) == 0 {
```

Comment:
You should not directly access v.Col, must MustTCol

Reply:
ok